build/source/driver/multi_driver.f90

```diff
@@ -709,6 +741,12 @@ program multi_driver
 
   ! loop through HRUs
   do iHRU=1,gru_struc(iGRU)%hruCount
+   
+   ! This line is used for debugging 
+   !write(*,'(4(A5,I10))') 'STEP',istep,'GRU',gru_struc(iGRU)%gru_id,'HRU',gru_struc(iGRU)%hruInfo(iHRU)%hru_id,'TYPE',typeStruct%gru(iGRU)%hru(iHRU)%var(iLookTYPE%vegTypeIndex)
+
+   ! write the forcing data to the model output file
+   call writeForce(fileout,forcStruct%gru(iGRU)%hru(iHRU),iHRU,jstep,err,message); call handle_err(err,message)
```

Comment:
Is it correct to pass `iHRU` (i.e., the local HRU index)? Consider the case where there is one HRU per GRU -- `iHRU` will always be equal to 1, and will be overwritten for subsequent GRUs.